api/v1alpha1/oadp_types.go

```diff
@@ -146,10 +146,29 @@ type ResticConfig struct {
 	PodConfig *PodConfig `json:"podConfig,omitempty"`
 }
 
+type NodeAgentConfig struct {
+	// Embedding ResticConfig
+	// +kubebuilder:validation:Optional
```

Comment:
this and `+optional` are the same? if yes, we could just one for standardization

Reply:
It is, looking at the: https://book.kubebuilder.io/reference/markers/crd-validation.html

```
// +kubebuilder:validation:Optional
specifies that this field is optional, if fields are required by default.

// +optional
specifies that this field is optional, if fields are required by default.
```

Reply:
changed to be consistent